### PR TITLE
Add related links to content stubs

### PR DESCRIPTION
**Pull request by jessicaschilling**
- Adds related links and associated GitHub issues to stub pages
- Pulls the preface content for issueNum in ContentStatus.vue so we can use full links there

This is one of two remaining pieces to close https://github.com/ipfs/docs/issues/342. The other is to remove "help us improve" link in the footer.

### CODE REVIEW

**Comment by jessicaschilling** [comment]
Pages affected:

- IPFS Glossary
- Usage ideas and examples
- The InterPlanetary Linked Data (IPLD) model
- The libp2p protocol
- IPFS Gateway
- History of the IPFS project
- Bitswap
- Immutability

File: docs/.vuepress/theme/global-components/ContentStatus.vue [hunk 1]
```diff
@@ -6,11 +6,7 @@
       </div>
       <h2>{{ title }}</h2>
       <div v-if="issueNum" class="content-status-status">
-        <a
-          target="_blank"
-          :href="`https://github.com/${repo}/issue/${issueNum}`"
-          >Check the status</a
-        >
+        <a target="_blank" :href="`${issueNum}`">Check the status</a>
```
Comment (cwaring) [inline docs/.vuepress/theme/global-components/ContentStatus.vue]:
```suggestion
        <a target="_blank" :href="issueUrl">Check the status</a>
```

File: docs/essentials/bitswap.md [hunk 2]
```diff
@@ -1,14 +1,13 @@
 ---
 title: Bitswap
 sidebarDepth: 0
-issueNum: 333
+issueNum: https://github.com/ipfs/docs/issues/85
```
Comment (cwaring) [inline docs/essentials/bitswap.md]:
```suggestion
issueUrl: https://github.com/ipfs/docs/issues/85
```

File: docs/project/history.md [hunk 3]
```diff
@@ -1,8 +1,14 @@
 ---
 title: History
 sidebarDepth: 0
+issueNum: https://github.com/ipfs/website/issues/352
```
Comment (cwaring) [inline docs/project/history.md]:
```suggestion
issueUrl: https://github.com/ipfs/website/issues/352
```

File: docs/essentials/usage-ideas-examples.md [hunk 4]
```diff
@@ -1,8 +1,14 @@
 ---
 title: Usage ideas & examples
 sidebarDepth: 0
+issueNum: https://github.com/ipfs/docs/issues/387
```
Comment (cwaring) [inline docs/essentials/usage-ideas-examples.md]:
```suggestion
issueUrl: https://github.com/ipfs/docs/issues/387
```

File: docs/essentials/libp2p.md [hunk 5]
```diff
@@ -1,8 +1,14 @@
 ---
 title: libp2p
 sidebarDepth: 0
+issueNum: https://github.com/ipfs/docs/issues/388
```
Comment (cwaring) [inline docs/essentials/libp2p.md]:
```suggestion
issueUrl: https://github.com/ipfs/docs/issues/388
```

File: docs/essentials/ipld.md [hunk 6]
```diff
@@ -1,8 +1,13 @@
 ---
 title: IPLD
 sidebarDepth: 0
+issueNum: https://github.com/ipfs/docs/issues/86
```
Comment (cwaring) [inline docs/essentials/ipld.md]:
```suggestion
issueUrl: https://github.com/ipfs/docs/issues/86
```

File: docs/essentials/ipfs-gateway.md [hunk 7]
```diff
@@ -1,8 +1,15 @@
 ---
 title: IPFS Gateway
 sidebarDepth: 0
+issueNum: https://github.com/ipfs/docs/issues/93
```
Comment (cwaring) [inline docs/essentials/ipfs-gateway.md]:
```suggestion
issueUrl: https://github.com/ipfs/docs/issues/93
```

File: docs/essentials/immutability.md [hunk 8]
```diff
@@ -1,8 +1,14 @@
 ---
 title: Immutability
 sidebarDepth: 0
+issueNum: https://github.com/ipfs/docs/issues/386
```
Comment (cwaring) [inline docs/essentials/immutability.md]:
```suggestion
issueUrl: https://github.com/ipfs/docs/issues/386
```

File: docs/essentials/glossary.md [hunk 9]
```diff
@@ -1,8 +1,13 @@
 ---
 title: Glossary
 sidebarDepth: 0
+issueNum: https://github.com/ipfs/docs/issues/56
```
Comment (cwaring) [inline docs/essentials/glossary.md]:
```suggestion
issueUrl: https://github.com/ipfs/docs/issues/56
```

**Pull request merged**